src/Lecture3.hs

```diff
@@ -34,6 +34,9 @@ module Lecture3
     , apply
     ) where
 
+{- | If you need to import libraries, do it after this line ... -}
+
+{- | ... and before this line. Otherwise the test suite might fail -}
```

Comment:
I think we need to have such a comment only in this file. Because only this module has the `$setup` part that needs to happen after imports. The rest are fine 🙂 

Also, I've added some arrows to attract more attention 👀 
```suggestion
{- VVV If you need to import libraries, do it after this line ... VVV -}

{- ^^^ ... and before this line. Otherwise, the test suite might fail ^^^ -}
```

Reply:
My reasoning for adding them to all files was:

1. Instructional: show people where it is customary to add imports.
2. Non-spoilery: I think `Lecture3.hs` is the first set of exercises where importing a library might make it easier, but no need to give that away?

Like the arrows ;-)

Reply:
@bulters Makes sense 🙂 I agree with that. In that case, if you could add arrows to all newly introduced sections, happy to merge 👍🏻

Reply:
Done.